src/test/e2e/migrator.e2e.test.ts

```diff
@@ -0,0 +1,40 @@
+import { getDbConfig } from './helpers/database-config';
+import { createTestConfig } from '../config/test-config';
+import { getInstance } from 'db-migrate';
+import { Client } from 'pg';
+import { IDBOption } from 'lib/types';
+
+async function initSchema(db: IDBOption): Promise<void> {
+    const client = new Client(db);
+    await client.connect();
+    await client.query(`DROP SCHEMA IF EXISTS ${db.schema} CASCADE`);
+    await client.query(`CREATE SCHEMA IF NOT EXISTS ${db.schema}`);
+    await client.end();
+}
```

Comment:
This is copied from unleash-enterprise repo and probably similar code exists in Unleash but because of time I keep it here, but anyone should feel free to contribute to this PR and make it better.